specification/ai/cspell.yaml

```diff
@@ -0,0 +1,114 @@
+# This file configures spell checking. Items in "words" were initially populated
+# with words that might be spelling errors. Review these words and take
+# appropriate action. For more information, see: https://aka.ms/ci-fix#spell-check
+
+# Spell checking is not case sensitive
+# Keep word lists in alphabetical so the file is easier to manage
+version: '0.2'
+import:
+  - ../../cspell.yaml
+words:
+  - acrad
+  - agatston
+  - apim
+  - ascvd
+  - autogen
+  - azurehealthinsightsradiologyinsights
+  - barcodes
+  - birads
+  - cancerprofiling
+  - ceus
+  - clinicaltrials
+  - codabar
+  - codeable
+  - computervision
+  - cusick
+  - dicom
+  - documentintelligence
+  - dynamicpersongroups
+  - extracolonic
+  - facelist
+  - facelists
+  - fhir
+  - findsimilars
+  - frax
+  - headwear
+  - healthinsights
+  - hnpcc
+  - imageanalysis
+  - interventional
+  - kellgren
+  - largefacelists
+  - largepersongroups
+  - loinc
+  - mednax
+  - nonstarted
+  - notstarted
+  - officedocument
+  - onco
+  - oncophenotype
+  - openai
+  - openxmlformats
+  - persistedfaces
+  - persondirectory
+  - persongroup
+  - persongroups
+  - presentationml
+  - radiologyinsights
+  - realface
+  - reslution
+  - retroperitoneal
+  - rrggbb
+  - smartcrops
+  - snomed
+  - spoofface
+  - spreadsheetml
+  - stepis
+  - subpackage
+  - supress
+  - tcgc
+  - telehealth
+  - tonnis
+  - trialmatcher
+  - tyrer
+  - umls
+  - unixtime
+  - upca
+  - upce
+  - valueset
+  - valuesets
+  - wordprocessingml
+overrides:
+  - filename: >-
+      **/specification/ai/data-plane/DocumentIntelligence/**/DocumentIntelligence.json
+    words:
+      - barcodes
+      - codabar
+      - documentintelligence
+      - heif
+      - presentationml
+      - rrggbb
+      - spreadsheetml
+      - upca
+      - upce
+  - filename: '**/specification/ai/data-plane/Face/**/*.json'
+    words:
+      - headwear
+      - realface
+      - spoofface
+  - filename: '**/specification/ai/data-plane/HealthInsights/stable/**/openapi.json'
+    words:
+      - acrad
+      - agatston
+      - ascvd
+      - birads
+      - ceus
+      - cusick
+      - extracolonic
+      - frax
+      - hnpcc
+      - kellgren
+      - mednax
+      - tonnis
+      - tyrer
+
```

Comment:
Example overrides. The `>-` chomping is done automatically by `js-yaml`. It can probably be configured not to chomp.

Reply:
Fixed